packages/web-forms/src/themes/2024-light/theme.scss

```diff
@@ -23,6 +23,7 @@ $emphasis-high: $textColour;
 :root {
 	--primary-50: #{$primary50};
 	--text-color: #{$textColour};
+	--secondary-text-color: #{$hintText};
```

Comment:
I'm curious: why introduce this new variable rather than the one we referenced that already exists? I would imagine this makes it more challenging to adopt PrimeVue tokens. I wasn't sure where the referenced `$hintText` came from, and apparently it's been defined for quite a while without reference. But it doesn't match any existing styles I can find.

Not worth another round trip unless you want to use `--text-color-secondary` as discussed (which does actually match the designs I can find in Figma). Up to you!

Reply:
Changed. I didn't realize `--text-color-secondary` is already defined by the theme. 

Aly had given me those colors on  [Jun 14th at 4:17 PM]
ODK blue #3E9FCC
Light blue #E9F8FF
Light gray #CBCACC
Text colour #1B1B1F
Hint text #6D6D6D
Error text #B3261E
Error light #FFEDEA